js/index.js

```diff
@@ -1 +1,68 @@
-// Your code goes here
+
+const navContainer = document.querySelector(".nav-container");
+const logoHeading = document.querySelector(".logo-heading");
+const nav = document.querySelectorAll("nav a");
+const h2 = document.querySelectorAll(".text-content h2");
+
+navContainer.addEventListener('click', () => {
+    logoHeading.style.color = 'red';
+});
+
+
+
+for(let i=0; i<nav.length; i++) {
+    nav[i].addEventListener('mouseover', (e) => {
+        console.log('Mouse is over link!');
+        e.target.style.color = 'green';
+    });
+}
+
+for(let i=0;i<nav.length;i++) {
+    nav[i].addEventListener('mouseleave', (e) => {
+        console.log('Mouse is out!');
+        e.target.style.color = '';
+    })
+}
+
+for(let i=0; i<nav.length; i++) {
```

Comment:
You have some simple errors here that are keeping your logic from working:

```
for(let i=0; i<h2.length; i++) {
    h2[i].addEventListener('dblclick', (e) => {
        e.target.style.style = "font-size: 2rem";
    });
};
```

1. You are looping through `nav.length` rather than `h2.length`.
2. You had `h2.addEventListener` rather than `h2[i].addEventListener`.